### PR TITLE
add changelog changes for Grid-accepting-dataframe

### PR DESCRIPTION
**Pull request by Kully**

### CODE REVIEW

**Comment by Kully** [comment]
Man, I keep forgetting the CHANGELOG...
Regarding the now closed-and-merged https://github.com/plotly/plotly.py/pull/675#pullrequestreview-18693158

@theengineear can you review quickly?

Also, is this the proper protocol with version numbers? - do we only bump the version number right before we know we are going to update the `pip package`, so that it grabs the new version number?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
```
Comment (theengineear) [inline CHANGELOG.md]: 👍 yup putting this under unreleased is correct. When you do a version bump, you change this to reflect the version number. When you're *sure* it's up on PyPI, you can set the date for the release and create a new Git Tag/Release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
...
+
+### Updated
```
Comment (theengineear) [inline CHANGELOG.md]: We haven't done a good job of this in the past, but let's not use the term `Updated`. [The guide](http://keepachangelog.com/en/0.3.0/) we follow suggests only using:

* `Added`
* `Changed`
* `Deprecated`
* `Removed`
* `Fixed`
* `Security`

You should use `Changed` here.

**Comment by Kully** [comment]
@theengineear all 💃 ?

**Comment by theengineear** [comment]
💃

**Pull request merged**